xarray/backends/zarr.py

```diff
@@ -271,6 +272,14 @@ class ZarrStore(AbstractWritableDataStore):
     def open_group(cls, store, mode='r', synchronizer=None, group=None,
                    writer=None):
         import zarr
+        min_zarr = '2.2'
+
+        if LooseVersion(zarr.__version__) < min_zarr:
```

Comment:
Can you add `# pragma: no cover` to indicate that this clause shouldn't be included in coverage calculations?

Reply:
Sure thing, hope I did it right!